src/webgpu/util/texture/layout.ts

```diff
@@ -107,7 +107,7 @@ export function getTextureSubCopyLayout(
   );
   assert(
     copySize_.width % blockWidth === 0 && copySize_.height % blockHeight === 0,
-    'copySize must be a multiple of the block size'
+    `copySize (${copySize_.width},${copySize_.height}) must be a multiple of the block size (${blockWidth},${blockHeight})`
```

Comment:
nit: you can tag `() =>` on the front here to avoid actually computing the message string unless it's needed. I assume this is probably useful for performance sometimes though I've never tested it.

Reply:
Done.